src/Razor/Microsoft.AspNetCore.Razor.Tools/test/RewriteCssCommandTest.cs

```diff
@@ -75,6 +75,40 @@ public void HandlesSpacesAndCommentsWithinSelectors()
 ", result);
         }
 
+        [Fact]
+        public void RespectsDeepCombinator()
```

Comment:
It will be interesting to have a case with multiple instances of `::deep`, something like `.a ::deep .b ::deep .c`

Reply:
Added test